doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,298 @@
+
+!listing lfr/heterogeneous_single_assembly_3D/cross_section/Step2/Fuel/LFR_127Pin_Fuel_1D_9g.mcc3.sh start=&control1d end=/ include-end=True
+
+For 1D geometry, there should be 29 cross section regions as indicated by `i_number_region=29`, which means that the number of elements of `i_mesh` (the number of fine-mesh in each cross section region), `r_location` (coordinates of each cross section region from center to periphery in a unit of $cm$ determined to preserve each material volume), and `i_composition` (index of composition assigned to each cross section region) must be 29. Composition indices from 2 to 8 are fuel corresponding to hexagonal rings 1 to 7, 1 is lead inside the duct, 9 is clad, 10 is duct, and 11 is lead outside the duct, as shown in the `material` block. For 1D cylindrical and slab geometries, only the white boundary condition is used.
+
+!listing lfr/heterogeneous_single_assembly_3D/cross_section/Step2/Fuel/LFR_127Pin_Fuel_1D_9g.mcc3.sh start=&geometry end=/ include-end=True
+
+In the `material` block, 7 different materials are defined for the fuel regions of 7 hexagonal rings. Even though the composition is the same, different isotope names are assigned to the same isotope to generate cross sections separately. On the other hand, only a single material is defined for each of lead, clad and duct regions, which means that only the average values over multiple cross section regions are generated per material. `i_externalspectrum(1)=6` means that the TWODANT spectrum of Zone 6 is superimposed to cross section region-wise spectrum for condensation. The index of `i_externalspectrum`, which is 1 here, is a special rule for a heterogeneous MC2-3 calculation that the superposition is applied to all cross section regions.
+
+Suffices in all compositions need special attention since it is directly related to how the ISOTXS file will be converted to the ISOXML file. When the resulting ISOTXS file that contains all isotopes in all compositions without composition IDs is converted to the ISOXML file, the last single character in the user-defined name is used to differentiate compositions and store cross sections of different compositions in different library IDs. Library IDs are increasing in the alphabetical order of the last singe character, from the starting value of $1$. In this MC2-3 input, eleven compositions are differentiated by the characters from A to K and the library IDs of them in the ISOXML file to be converted in a later stage are from $1$ to $11$. If a isotope name excluding the last single character is not the standard isotope name defined in ISOXML, the pre-fix `pseudo_` is attached. Otherwise, the isotope name is used as is.
```

Comment:
```suggestion
Suffices in all compositions need special attention since it is directly related to how the ISOTXS file will be converted to the ISOXML file. When the resulting ISOTXS file that contains all isotopes in all compositions without composition IDs is converted to the ISOXML file, the last single character in the user-defined name is used to differentiate compositions and store cross sections of different compositions in different library IDs. Library IDs are increasing in the alphabetical order of the last single character, from the starting value of $1$. In this MC2-3 input, eleven compositions are differentiated by the characters from A to K and the library IDs of them in the ISOXML file to be converted in a later stage are from $1$ to $11$. If a isotope name excluding the last single character is not the standard isotope name defined in ISOXML, the pre-fix `pseudo_` is attached. Otherwise, the isotope name is used as is.
```